br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -148,6 +148,20 @@ const (
 		ORDER BY raw_key;
 	`
 
+	selectDataConflictKeysReplace = `
```

Comment:
same problem as https://github.com/pingcap/tidb/pull/45926#discussion_r1302463762

Reply:
I think here the situation is different than https://github.com/pingcap/tidb/pull/45926#discussion_r1302463762

We need to first use `rawKey` to determine `mustKeepKvPairs` for this `rawKey`, then iterate all the KV with this `rawKey`, in the iteration this corresponding `mustKeepKvPairs` is needed. If we just use one SQL query, we may need to store all `mustKeepKvPairs`, which is not efficient.